pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/JsonAsyncHttpPinotClientTransport.java

```diff
@@ -70,7 +68,7 @@ public JsonAsyncHttpPinotClientTransport() {
   }
 
   public JsonAsyncHttpPinotClientTransport(Map<String, String> headers, String scheme, String extraOptionString,
-      boolean useMultistageEngine, @Nullable SSLContext sslContext, ConnectionTimeouts connectionTimeouts,
+      boolean useMultistageEngine, @Nullable SslContext sslContext, ConnectionTimeouts connectionTimeouts,
```

Comment:
since this is public API we might need to keep supporting both javax.net.ssl.SSLContext and io.netty.handler.ssl.SslContext and mark the former with deprecated annotation.

Reply:
updated

Reply:
:+1: